lib/milc_interface.cpp

```diff
@@ -1426,8 +1426,9 @@ void qudaInvertMsrc(int external_precision, int quda_precision, double mass, Qud
 
   // return the number of iterations taken by the inverter
   *num_iters = invertParam.iter;
```

Comment:
I believe MILC expects the aggregate number of iterations as if it wasn't a batch/block solve, but I forget. I'll also scope out how it expects the residuals as part of the work I'm doing on the MILC side and sort this out

Reply:
Note to self, can confirm that is the case: https://github.com/lattice/milc_qcd/blob/develop/generic_ks/d_congrad5_fn_milc.c#L409-L417

Reply:
Resolved